htdocs/mrp/index.php

```diff
@@ -147,6 +148,71 @@
 
 print '<br>';
 
+$sql = "SELECT bom.rowid, bom.ref, bom.label ";
+$sql .= " FROM ".MAIN_DB_PREFIX."bom_bom AS bom, ".MAIN_DB_PREFIX."bom_bomline AS blin, ";
+$sql .= MAIN_DB_PREFIX."entrepot AS entpt, ".MAIN_DB_PREFIX."product_stock AS stk";
+$sql .= " WHERE bom.rowid = blin.fk_bom";
+$sql .= " AND blin.fk_product = stk.fk_product";
+$sql .= " AND stk.fk_entrepot = entpt.rowid";
+$sql .= " AND entpt.statut = 1";
+$sql .= " AND bom.entity = ".getEntity('bom');
+$sql .= " AND bom.status = 1";
+$sql .= " AND bom.rowid NOT IN (";
+$sql .= " SELECT b.rowid";
+$sql .= " FROM ".MAIN_DB_PREFIX."bom_bom AS b";
+$sql .= " WHERE b.entity = ".getEntity('bom');
+$sql .= " AND b.rowid IN (";
+$sql .= " SELECT fk_bom";
+$sql .= " FROM ".MAIN_DB_PREFIX."bom_bomline AS bl";
+$sql .= " LEFT JOIN `llx_product_stock` AS st";
+$sql .= " ON";
+$sql .= " bl.fk_product = st.fk_product";
+$sql .= " WHERE st.fk_product IS NULL OR bl.qty > (";
+$sql .= " SELECT SUM(s.reel)";
+$sql .= " FROM ".MAIN_DB_PREFIX."product_stock AS s,";
+$sql .= " ".MAIN_DB_PREFIX."entrepot AS ent";
+$sql .= " WHERE s.fk_product = bl.fk_product";
+$sql .= " AND s.fk_entrepot = ent.rowid";
+$sql .= " AND ent.statut = 1";
+$sql .= " AND ent.entity =".getEntity('bom');
```

Comment:
Use 
$sql .= " AND ent.entity IN (".getEntity('warehouse').")";

Reply:
@eldy @inoveaconseil 

$sql .= " AND ent.entity IN (".getEntity('stock').")";